templates/taxi/car_list.html

```diff
@@ -7,6 +8,12 @@ <h1>
       Create
     </a>
   </h1>
+
+  <form action="" class="form-inline">
+    {{ search_field|crispy }}
+    <input type="submit" value="🔍" class="btn btn-secondary">
+  </form>
```

Comment:
Please create a separate template for the search form and include it where needed.